modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/Filters.tsx

```diff
@@ -269,12 +269,12 @@ function Filters({
 
 		const responseJSON = await response.json();
 
-		const storedFDSFiltersOrder = responseJSON?.fdsFiltersOrder;
+		const storedFDSFiltersOrder = responseJSON?.filtersOrder;
```

Comment:
we can also take this opportunity to remove `FDS` from here as well: `storedFiltersOrder`